test/integration/eu-west-1/k8s-qa18/services/terragrunt.hcl

```diff
@@ -74,4 +73,21 @@ inputs = {
 
   kiam_chart_version = "5.9.0" # 5.10.0 includes https://github.com/uswitch/kiam/pull/427, but image does not - breaks WhitelistRouteRegexp/AllowRouteRegexp
 
+  # --------------------------------------------------
+  # Monitoring namespace
+  # --------------------------------------------------
+
+  monitoring_namespace_deploy = true
+
+  # --------------------------------------------------
+  # Metrics server
+  # --------------------------------------------------
+
+  monitoring_metrics_server = true
```

Comment:
Chart version missing to match production

Reply:
Are we matching production in the QA? I thought the point was also to test upstream changes, maybe I'm wrong here. I'll just apply the chart version for now.